scripts/exec-tests.sh

```diff
@@ -368,9 +368,9 @@ echo "list of tests: $tests"
 for testname in $tests; do
   # Defer uninstall till after other tests have been run.
   if [ "$testname" != "uninstall" ]; then
-    if [ -d "$TESTDIR/tests/$testname" ]; then
+    if [ -d "$TESTDIR/tests/lvm/$testname" ]; then
```

Comment:
what is the plan for tests which are not `lvm` tests?

Reply:
support for other engine tests will be added once tests are implemented for other engine